test/runtests.jl

```diff
@@ -212,7 +212,6 @@ end
             @info "test3" i=i j=2^i dd=rand(10).-0.1*i hh=data_tuple log_step_increment=0
         end
     end
-
     @test TensorBoardLogger.step(logger) == 10
```

Comment:
I don't know. were their other failures?

Reply:
figured it out. it was happening whenever `testimages` downloaded data inside the do block, it would somehow call `handle_message` function and cause step to increment